bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java

```diff
@@ -365,7 +377,7 @@ public Void call() throws Exception {
                         } else {
                             logger.info(
                                     "Not delegating command '{}' for item '{}' to handler for channel '{}', "
-                                            + "because thing is not initialized (must be in status ONLINE or OFFLINE).",
+                                            + "because thing is not initialized (must be in status UNKNOWN, ONLINE or OFFLINE).",
```

Comment:
This should probably be changed to "handler is not initialized", see https://github.com/eclipse/smarthome/pull/2599#discussion_r91108335.